index.html

```diff
@@ -34,7 +34,7 @@
 								<li><a href="#">Join</a></li>
 								<li><a href="#">Sponsor</a></li>
 								<li><a href="#">News</a></li>
-								<li><a href="#">CC Campaign</a></li>
+								<li><a href="#" class="campain">CC Campaign</a></li>
```

Comment:
- [ ] Please as the last reviewer said implement the hovering effects on the buttons 

| Your design | 
| ------------- |
| ![image](https://user-images.githubusercontent.com/74744787/163464502-35cc9d10-6f98-4dfe-bf13-6a2be04f2e72.png) |

Reply:
Hello @MeqdamAlqudah please kindly note that there are no buttons, that is a menu item with  border and I have implemented it just like the design, also there is a hover effect on my nav elements